### PR TITLE
Added more WMDE github repositories to the list

### PR DESCRIPTION
**Pull request by manicki**
I maybe could/should have made a separate PR per repo or so, to allow some discussion (I suspect some repos might be debatable).
I'll submit another PR with some links, that are not that clear to me, shortly.

### CODE REVIEW

**Comment by datn** [comment]
Recommend removing wikiba.se as it will soon (in the geological time of bureaucracy) be deprecated

**Comment by manicki** [comment]
> Recommend removing wikiba.se as it will soon (in the geological time of bureaucracy) be deprecated

done!

**Comment by lucaswerkmeister** [comment]
> Recommend removing wikiba.se as it will soon (in the geological time of bureaucracy) be deprecated

Hm, I don’t think it makes sense to remove that while at the same time adding [mediawiki-doctrine-connection](https://github.com/wmde/mediawiki-doctrine-connection) (experiment during the new termstore work, will presumably never be used) or [WikidataApiGem](https://github.com/wmde/WikidataApiGem) (maybe used in our Ruby browser tests? but no changes since 2019), not to mention the things from #3.

**Comment by Ladsgroup** [comment]
> > Recommend removing wikiba.se as it will soon (in the geological time of bureaucracy) be deprecated
> 
> Hm, I don’t think it makes sense to remove that while at the same time adding [mediawiki-doctrine-connection](https://github.com/wmde/mediawiki-doctrine-connection) (experiment during the new termstore work, will presumably never be used) or [WikidataApiGem](https://github.com/wmde/WikidataApiGem) (maybe used in our Ruby browser tests? but no changes since 2019), not to mention the things from #3.

I think `mediawiki-doctrine-connection` should be just archived?

**Comment by datn** [comment]
> > Recommend removing wikiba.se as it will soon (in the geological time of bureaucracy) be deprecated
> 
> Hm, I don’t think it makes sense to remove that while at the same time adding [mediawiki-doctrine-connection](https://github.com/wmde/mediawiki-doctrine-connection) (experiment during the new termstore work, will presumably never be used) or [WikidataApiGem](https://github.com/wmde/WikidataApiGem) (maybe used in our Ruby browser tests? but no changes since 2019), not to mention the things from #3.

OK, feel free to add it and then remove it in a few weeks/months.

File: README.md [hunk 1]
```diff
@@ -38,6 +38,13 @@ WIP team site
 - https://github.com/wmde/Number
 - https://github.com/wmde/Time
 - https://github.com/wmde/git-filter-repo-docker-action
+- https://github.com/wmde/vuex-helpers
+- https://github.com/wmde/lib-version-check
+- https://github.com/wmde/hamcrest-html-matchers
+- https://github.com/wmde/mediawiki-doctrine-connection
+- https://github.com/wmde/WikidataApiGem
+- https://github.com/wmde/wbs_propertypairs
+- https://github.com/wmde/prototype-wikibase-graphql-api
```
Comment (tzhelyazkova) [inline README.md:47]: not sure this fits the intention of the list. It's very much a prototype and if it has served its purpose it should be archived even.

**Review by tzhelyazkova — COMMENTED** [review]
While you're at it, can you add Cognate to the list of extensions? We seem to support it as we had a Cognate task on camp recently.

File: README.md (hunk 1 above)
Comment (addshore) [reply README.md:47]: I had the same thought over at https://github.com/wmde/wikidata-wikibase-team/pull/3#issuecomment-800511977 regarding some of the repositories there.

As this is "just a list" for now I think we could include them.
In a future iteration I think this list should include more categorizations of the repos / taggings.

**Comment by addshore** [comment]
> > > Recommend removing wikiba.se as it will soon (in the geological time of bureaucracy) be deprecated
> > 
> > 
> > Hm, I don’t think it makes sense to remove that while at the same time adding [mediawiki-doctrine-connection](https://github.com/wmde/mediawiki-doctrine-connection) (experiment during the new termstore work, will presumably never be used) or [WikidataApiGem](https://github.com/wmde/WikidataApiGem) (maybe used in our Ruby browser tests? but no changes since 2019), not to mention the things from #3.
> 
> I think `mediawiki-doctrine-connection` should be just archived?

+1 I think we should just archive this and remove it from the list being added here

**Comment by addshore** [comment]
> While you're at it, can you add Cognate to the list of extensions? We seem to support it as we had a Cognate task on camp recently.

I added Cognate and InterwikiSorting in https://github.com/wmde/wikidata-wikibase-team/commit/c3e152151d0ed82951a04e02eab6493d0b73ed87

File: README.md (hunk 1 above)
Comment (manicki) [reply README.md:54]: I was about to remove it from the list, but then @addshore suggested to include "all the things". Keeping in the list, maybe the repo gets archived meanwhile and then will drop off the list.

**Comment by addshore** [comment]
Going to merge this as is, but super happy to keep receiving PRs etc tweaking things, adding things, removing things, adding a "prototype" section or something similar.

**Pull request merged**